apps/frontend/src/components/global/ExportSplunkModal.vue

```diff
@@ -45,6 +45,11 @@
                   Are you sure that you have configured it properly?
                 </span>
                 <br />
+                <span
```

Comment:
<img width="681" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/4d3a8a47-54ae-4d05-8f15-db4c2454e4b6">

now that we're putting actual content into there instead of a link out, we should probably make it left aligned instead of centered.  this is for both the splunk tab on the load modal as well as in the export.